app/views/layouts/overdue.html.erb

```diff
@@ -0,0 +1,99 @@
+<% content_for(:content) do %>
```

Comment:
This new layout was mostly modeled after the my facilities layout, which has a similar sidebar.